example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
+            end
+            if ENV['REPO_NAME'] == "on-tasks"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-tasks/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-tasks/" 
+            end
+            if ENV['REPO_NAME'] == "on-core"
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-taskgraph/node_modules/on-core/"  
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-http/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-tftp/node_modules/on-core/" 
+              target.vm.synced_folder "#{ENV['WORKSPACE']}/build", "/home/vagrant/src/on-dhcp-proxy/node_modules/on-core/" 
+            end
+          end
+        end
+        
+        if ENV['CONFIG_DIR']
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/opt/monorail/"
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/opt/onrack/etc/"
+          target.vm.synced_folder "#{ENV['WORKSPACE']}/#{ENV['CONFIG_DIR']}", "/home/vagrant/opt/monorail/"
+        end
 
+        if ENV['TEST_GROUP']		
+          if ENV['TEST_GROUP'] =~  /esxi-5-5-min-install.v2.0.test/ || ENV['TEST_GROUP'] =~  /esxi-5-5-max-install.v2.0.test/		
+            config.vm.provision "file", source: "./dhcpd.conf", destination: "~/dhcpd.conf"		
+            config.vm.provision "shell" do |s|		
+            s.inline = "cp /home/vagrant/dhcpd.conf /etc/dhcp"		
+            s.privileged = true		
+            end		
+          end		
+        end
+        if ENV['PYTHON_REPOS']
+          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
+          config.vm.provision "shell" do |s|		
+            s.inline = "apt-get install python-pip -y"		
+            s.privileged = true
+         end
+          list = ENV['PYTHON_REPOS'].split(' ')
+          list && list.each do |repo|
+            config.vm.provision "shell" do |s|		
+            s.inline = "cd /home/vagrant/src/#{repo} && pip install -r requirements.txt"		
+            s.privileged = true
+            end
+          end
+        end
+        
         # Specify optional interface to bridge when wanting to expose the instance to an
         # external network.  Useful for when discovering nodes externally.
         if ENV['BRIDGE_INTF']
-           target.vm.network "public_network", ip: "172.31.128.1", bridge: "#{ENV['BRIDGE_INTF']}"
+          target.vm.network "public_network", ip: "172.31.128.1", bridge: "em1"
         else
-           target.vm.network "private_network", ip: "172.31.128.1", virtualbox__intnet: "closednet"
+          target.vm.network "private_network", ip: "172.31.128.1", virtualbox__intnet: "closednet"
```

Comment:
Thanks for the heads up. I think the original Vagrantfile uses 3 spaces of indentation inside the if block and the one from on-build-config uses 2. Not sure if it matters?